.travis.yml

```diff
@@ -1,5 +1,68 @@
-language: node_js
-node_js:
-  - "node"
+sudo: false
+
+language: php
+
+notifications:
+  email:
+    on_success: never
+    on_failure: change
+
+branches:
+  only:
+    - master
+
```

Comment:
The above 3 lines will need to be removed won't they if we want Travis to run against each PR?